### DISCUSSION TITLE
How to disable automatic dark mode? (Mac)

### DISCUSSION THREAD

**Discussion by marchpane** [dbeaver/dbeaver#14728] [Q&A] [answered]
Dear Dbeaver enthusiasts,

is there a possibility to disable the automatic switch between dark mode and non dark mode? Unfortunately there is no option I can find who might disable this behaviour and in the evening it just looks terrible.

User Interface -> Apperance -> Enable theming is not checked. Even when I check it and select for example "Light" it looks like in the picture attached.

I would appreciate any help.

DBeaver 21.2.5.202111141807
OS: Monterey

<img width="1521" alt="Screen1" src="https://user-images.githubusercontent.com/89104892/143937970-d6f28bb9-5dec-4e48-8d58-1249c24f69b5.png">

**Comment by marchpane** [comment, marked as answer]
Update: With the newly released update this issue doesn't come up again as it seems (so far).

**Reply by ShadelessFox** [reply]
Yes, it was fixed in #13968.
